screens/Search/index.tsx

```diff
@@ -51,16 +62,16 @@ const FindFriends = () => {
   }
 
   const handleAddFriend = async (user: User) => {
-    if (isLoading || user.isFriend) return
+    if (isLoadingFriend[user.id] || user.isFriend) return
```

Comment:
The purpose of the "isLoading" was to ensure that no other request was sent while the first friend was still being created.